<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delTrafficDialog</class>
 <widget class="QDialog" name="delTrafficDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTrafficBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trafficEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="traffic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>班次：</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>出发日期：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="typeEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="type">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>类型：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
